x-pack/plugin/core/src/main/resources/fleet-files.json

```diff
@@ -10,8 +10,8 @@
   },
   "template": {
     "settings": {
-      "index.lifecycle.name": ".fleet-files-ilm-policy",
```

Comment:
Is the `fleet-file-data-ilm-policy.json` policy still needed?

Reply:
we will need the ILM policies for the full fix, if removing them prevents us from shipping that in 8.7.x I'd rather keep them there if possible?

Reply:
Can you expand a bit on what the full fix is?

Reply:
Sure, its not 100% as we only discovered this yesterday, but we will likely be looking to have kibana create the index templates at package install time, which will then refer to these ILM policies. For now removing ILM from the current index template means the feature still works.

Our issue at the moment is that we were trying to use one index template for all of our integrations that will be using this feature, but as each integration has a different alias, we aren't able to set the ilm `rollover_alias` on the index template, we thought setting it on the first write index works but we were wrong. 

Moving to kibana creating the index templates will mean we wont have to have an index template for each integration in elasticsearch repo.

I have an issue for the long term fix here https://github.com/elastic/kibana/issues/153504